lib/storage/src/content_manager/consensus_state.rs

```diff
@@ -917,4 +942,89 @@ mod tests {
         // Even when `max_size` is `0` this fn should return at least 1 entry
         assert_eq!(wal.entries(4, 5, Some(0)).unwrap().len(), 1)
     }
+
+    struct NoCollections;
```

Comment:
Where is this struct used?

Reply:
I must have been tired when writing this comment, I clearly see it used in `setup_storages` now :sleepy:

Reply:
No worries!